### DISCUSSION TITLE
pid and max31865 to control espresso temp

### DISCUSSION THREAD

**Discussion by hooijschuur** [arendst/Tasmota#20455] [Q&A]
Hello i try to control the heat of my espresso machine with tasmota.
but the json temp update frequenty of minimal 10 seconds looks a bit slow.
is there faster way to get the the temperture in the pid controller

**Comment by Wintespe** [comment]
It looks like the value of 

`    TIMEPROP_CYCLETIMES 60 // Cycle time seconds`

is too large.  Unfortunately, the value cannot be set in the console. To change it, you need to edit the file

 `    xdrv_48_timeprop.ino`

and complie Tasmota..
